### PR TITLE
feat: workflow diagrams

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -355,8 +355,37 @@ <h5>Sample Steel Workflow</h5>
...
               <figcaption>Sample Steel Workflow</figcaption>
             </figure>
-            
+            <p>
+              A new bridge in Pennsylvania is being built, but to construct it, the contractor needs steel beams, which requires their
+              preferred American steel beam manufacturer to purchase steel slabs.
+            </p>
+            <p>
+              A Mexican steel slab manufacturer has already produced these slabs, applied a decentralized identifier to each slab, and
+              sent the slabs for testing. During testing an inspector certified that the physical and chemical properties of the steel
```
Comment (TallTed) [inline docs/index.html:364]:
```suggestion
              sent the slabs for testing. During testing, an inspector certified that the physical and chemical properties of the steel
```

File: docs/index.html [hunk 2]
```diff
@@ -527,8 +556,48 @@ <h5>Sample E-Commerce Workflow</h5>
             <img src="./ecommerce-workflow.png" alt="Sample E-Commerce Workflow" style="width:100%">
             <figcaption>Sample E-Commerce Workflow</figcaption>
           </figure>
-          
-
+          <p>
+          A large and popular shoe manufacturer is ready to come out with their newest line of sneakers. First, they'll place an
+          order to a company in Anshan China to manufacture their new product line. Then, the Chinese company manufactures the
```
Comment (TallTed) [inline docs/index.html:561]:
```suggestion
          order to a company in Anshan, China, to manufacture their new product line. Then, the Chinese company manufactures the
```

**Comment by VladimirAlexiev** [comment]
@OR13 @TallTed @nissimsan These diagrams look nice.
Guess they are stolen from somewhere :-)

For making custom diagrams, have you guys considered PlantUML? 
Describe your diagrams in simple text.
It's a brilliant invention, very responsive dev, excellent features.

**Comment by OR13** [comment]
example

Example of a rendering of a workflow of related credential and presentations

**Comment by OR13** [comment]
good suggestion: https://plantuml.com/

I think we might consider a section on this kind of tooling as it related to "workflows" just to assist readers in finding tools in the future.

**Comment by nissimsan** [comment]
@VladimirAlexiev, no I made these myself. But flattering that you find them nice enough to think they were stolen. :D

**Comment by VladimirAlexiev** [comment]
@nissimsan @OR13 
All https://github.com/gs1/EPCIS/tree/master/Diagrams are automatically generated with my `rdfpuml` from actual Turtle examples. 

The neo4j diagrams are nice for exploration, but I think the above are a lot more readable.

Of course, if most of the payload is a tree not graph, it's a lot easier to use a tree layouter...

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**